src/components/MultiButtonWithSubQuery.tsx

```diff
@@ -0,0 +1,106 @@
+import { StackNavigationProp } from '@react-navigation/stack';
+import React from 'react';
+
+import { Button } from './Button';
+
+export type TSubQueryType = {
```

Comment:
"T" already means Type...so T...Type is a duplication. i will update while moving to types folder.